src/Inkluzitron/Services/PointsService.cs

```diff
@@ -69,12 +77,21 @@ private async Task<IUser> LookupUserAsync(ulong userId)
                 return null;
         }
 
-        public async Task<IReadOnlyList<GraphItem>> GetAllPointsAsync()
+        public async Task<IReadOnlyList<GraphItem>> GetUsersTotalPointsAsync()
```

Comment:
Trochu zde mám obavu, že to tahá celou tabulku

Reply:
To ti vůbec nerozumím. 

Leze z toho tento SQL dotaz:
```sql
SELECT "u"."Id", (
    SELECT COALESCE(SUM("d"."Points"), 0)
    FROM "DailyUserActivity" AS "d"
    WHERE "u"."Id" = "d"."UserId") AS "TotalPoints"
FROM "Users" AS "u"
ORDER BY (
    SELECT COALESCE(SUM("d0"."Points"), 0)
    FROM "DailyUserActivity" AS "d0"
    WHERE "u"."Id" = "d0"."UserId")
```

Nějak nevidím, co by na tom mělo být špatně/jak to udělat líp.

Reply:
Tru, jak tak nad tím přemýšlím, tak mě taky nenapadá lepší způsob. 🤔